.github/workflows/erlang.yml

```diff
@@ -1,10 +1,7 @@
 name: Erlang CI
 
 on:
-  push:
-    branches: [ develop-3.0 ]
-  pull_request:
-    branches: [ develop-3.0 ]
+  pull_request
```

Comment:
Sure you want to do it on each pull, even master branch... which has rebar and not rebar3?

Reply:
I'm not sure what the right configuration is, but I've reverted to make consistent with the other repos for now.